### ISSUE TITLE
How to test the model on COCO test-dev set 2017

### ISSUE THREAD

**Issue by Ozone-oo** [microsoft/human-pose-estimation.pytorch#152]
I want to test model on test-dev set, but the code pose_estimation/valid.py is only for testing on validation set (5000 images):

So, I have a question: How to test the model on COCO test-dev set 2017 to submit the json file to codalab server?
I'm looking forward from you. Thank you for your great work!

**Comment by desires19** [comment]
I have met the question too, so can you tell me how to solve this problem if you have solved it.
Thank u, my bro!

**Comment by Ozone-oo** [comment]
> I have met the question too, so can you tell me how to solve this problem if you have solved it.
> Thank u, my bro!

1、Download the test2017 dataset and image info.
2、Download the detection result on test-dev2017.
3、Modify the yaml file as follow
DATASET:
TEST_SET: 'test-dev2017'
TEST:
COCO_BBOX_FILE: 'data/coco/person_detection_results/COCO_test-dev2017_detections_AP_H_609_person.json'
USE_GT_BBOX: false
